src/components/creators/creator-card.tsx

```diff
@@ -0,0 +1,28 @@
+import Image from "next/image";
+
+export interface CreatorCardProps {
+    rank: number;
+    avatar: string;
+    name: string;
+    totalSales: number;
+}
+
+const CreatorCard = ({ rank, avatar, name, totalSales }: CreatorCardProps) => {
+    return (
+        <div className="w-60 flex flex-col items-center justify-center relative bg-[#3B3B3B] rounded-3xl p-5 mt-10">
```

Comment:
You can add color variables to the tailwind setup, so you don't need to write code of color anymore.

`bg-[#3B3B3B] -> bg-mineShaft`

You can find name of the color here - https://chir.ag/projects/name-that-color/#3B3B3B